website/docs/reference/resource-configs/hard-deletes.md

```diff
@@ -0,0 +1,113 @@
+---
+title: hard_deletes
+resource_types: [snapshots]
+description: "Use the `hard_deletes` config to control how deleted rows are tracked in your snapshot table."
+datatype: "{<dictionary>}"
+default_value: {ignore}
+id: "hard-deletes"
+sidebar_label: "hard_deletes"
+---
+
+Available from dbt v1.9 or with [Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
+
+
+<File name='snapshots/schema.yml'>
+
+```yaml
+snapshots:
+  - name: <snapshot_name>
+    config:
+      hard_deletes: 'ignore', 'invalidate', or 'new_record'
+```
+</File>
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +hard_deletes: "ignore", "invalidate", or "new_record"
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{{
+    config(
+        unique_key='id',
+        strategy='timestamp',
+        updated_at='updated_at',
+        hard_deletes='ignore', 'invalidate', 'new_record'
+    )
+}}
+```
+
+</File>
+
+
+## Description
+
+Use the `hard_deletes` configuration to track hard deletes by adding a new record when row become "deleted" in source. 
+Replaces the `invalidate_hard_deletes` config to give you more control on how to handle deleted rows from the source.
```

Comment:
I would remove this, and just have it called out when talking about the `invalidate` method